src/devicecontroller/DefaultDeviceController.ts

```diff
@@ -520,9 +548,24 @@ export default class DefaultDeviceController implements DeviceControllerBasedMed
         newDevice.stream = stream;
         newDevice.constraints = proposedConstraints;
       } else {
+        // TODO: remove check when the bug is fixed.
+        if (
+          kind === 'audio' &&
+          this.activeDevices[kind] &&
+          device &&
+          this.getDeviceIdStr(device) === 'default'
+        ) {
+          if (
+            this.activeDevices[kind].constraints &&
+            this.activeDevices[kind].constraints.audio &&
+            JSON.parse(JSON.stringify(this.activeDevices[kind].constraints.audio)).deviceId
+              .exact === 'default'
+          ) {
+            this.activeDevices[kind].stream.getAudioTracks()[0].stop();
```

Comment:
Why not use [this.releaseMediaStream](https://github.com/aws/amazon-chime-sdk-js/blob/master/src/devicecontroller/DefaultDeviceController.ts#L231)?

Reply:
updated